### ISSUE TITLE
bug in firefox window alignment

### ISSUE THREAD

**Issue by gutierri** [jcs/sdorfehs#7]
...both in relation to the existing windows and the screen.

it apparently adds a margin at x, y in addition to the border value (my current setting is 3 for border size).

I am attaching a screenshot to help.

obs: I have been using ratpoison with [this](https://github.com/gutierri/ratpoison/commit/ababbd82cca09c2854e5effa48feaf1041b25886) patch of yours and it works perfectly, less on sdorfehs.

![screenshot firefox on sdorfehs](https://user-images.githubusercontent.com/1111356/87351946-a5dc4600-c530-11ea-980e-3c4b114163c1.png)

**Comment by jcs** [comment]
I need your full config file to reproduce it.

**Comment by jcs** [comment]
Do you still see this with c94a4cc0?

**Comment by gutierri** [comment]
the problem still exists ...

a behavior that I noticed and that when removing the edges (set border 0), the Firefox frame fits the screen perfectly without additional margins.

Em 13 de jul de 2020, à(s) 23:29, joshua stein <notifications@github.com> escreveu:

**Comment by gutierri** [comment]
in the logs the output of the Y and X coordinates are:

```
sdorfehs:manage.c:763: debug: Resizing normal window 'Mozilla Firefox' to x:3 y:3 w:754 h:294¬
sdorfehs:globals.c:288: debug: Giving focus to 'Mozilla Firefox'¬
sdorfehs:events.c:761: debug: --- Handling PropertyNotify ---¬
sdorfehs:events.c:536: debug: atom: 251 (WM_STATE)¬
```

the expected is that it is X 0, Y 0, equal to the other windows.

from what I understand little of the old algorithm, it always adds 3 to the X and Y coordinates (captured via log). I opened a [PR](url) in WIP that fixed this problem. I took this patch from the already archived _jcs/ratpoison_ tree.

**Comment by jcs** [comment]
Fixed in 543576bbf7
